### PR TITLE
build: add release script to automate the tag process

### PR DESCRIPTION
**Pull request by Yuvalke**

### CODE REVIEW

File: app/dist_src/studio.ini [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [general]
 component.name = HTML5StudioV3 ;the application Name
-component.version = 3.12.3.rc2 ;similar to folder name
+component.version = 3.12.3.rc3 ;similar to folder name
```
Comment (yairans) [inline app/dist_src/studio.ini:3]: dont you need to make the version by template e.g. {{VERSION}}?

File: app/studio.ini [hunk 2]
```diff
@@ -1,9 +1,9 @@
 [general]
 component.name = HTML5StudioV3 ;the application Name
-component.version = 3.12.3.rc2 ;similar to folder name
+component.version = 3.12.3.rc3 ;similar to folder name
```
Comment (yairans) [inline app/studio.ini:3]: same

File: scripts/release.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+read -p "Enter Tag Number without prefix v: "  tag
+
+PREV_TAG=$(git describe --tags)
```
Comment (yairans) [inline scripts/release.sh:4]: means you take the last?

File: app/dist_src/studio.ini (hunk 1 above)
Comment (Yuvalke) [reply app/dist_src/studio.ini:3]: After first commit, it won't be the same

File: scripts/release.sh (hunk 3 above)
Comment (Yuvalke) [reply scripts/release.sh:4]: Correct, I check the latest

Comment (yairans) [reply scripts/release.sh:4]: not so safe i think

**Review by yairans — CHANGES_REQUESTED** [review]

**Review by yairans — APPROVED** [review]

**Pull request merged**